README.md

```diff
@@ -44,9 +44,18 @@ Installation with CocoaPods
 
    If your test target is Objective-C only use `iOSSnapshotTestCase/Core` instead, which doesn't contain Swift support.
 
-   Replace "Tests" with the name of your test project.
+Installation Carthage
```

Comment:
How about instead of adding "Installation Carthage" as another top level header let's rename "Installation with CocoaPods" to "Installation" (or "How to install") and then have "Cocoapods" and "Carthage" as subheaders.

Having "Installation with Carthage", "Installation Carthage" and "Setup Test Scheme" could be a bit confusing since they all apply to how to install the library.

Reply:
Agreed,

Will be better the way you proposed.

Tks =)